Includes/timeMenu.cpp

```diff
@@ -10,6 +10,7 @@ TimeItem::TimeItem(MenuNode* parent, std::string const& label) : MenuItem(parent
 }
 
 void TimeItem::execute(Menu* menu) {
```

Comment:
I believe the way this is done elsewhere is to remove the argument name (compare `(Menu* menu)` and `(Menu*)`). Same goes for the FTP stuff. :)

Reply:
I could've sworn that I tried that initially and was still getting the warning but just confirmed that it does fix it, sorry! Changed here and elsewhere.